instaclustr/resource_cluster.go

```diff
@@ -710,6 +732,15 @@ func resourceClusterRead(d *schema.ResourceData, meta interface{}) error{
 	return nil
 }
 
+func stringInSlice(a string, list []string) bool {
```

Comment:
[desired] this function could go in the utils file (as dirty as that is, its in keeping with what we have)

Reply:
it amazes me how many useful functions go doesn't seem to have. It doesn't have a built in max function!

Agreed though, will do that.

Reply:
yup its very low coverage from this side of things